qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -119,13 +121,22 @@ def convert(
                     If a Tuple[ParameterExpression, ParameterExpression] or
                     List[Tuple[ParameterExpression, ParameterExpression]]
                     is given, then the 2nd order derivative of the operator is calculated.
-
+            aux_meas_op: The operator that the auxiliary qubit is measured with respect to.
+                For ``aux_meas_op = Z`` we compute 2Re[⟨(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉],
+                for ``aux_meas_op = -Y`` we compute 2Im[⟨(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉], and
+                for ``aux_meas_op = Z-1jY`` we compute 2⟨(dω⟨ψ(ω)|)O(θ)|ψ(ω).
```

Comment:
If this line is framed as a code fragment then it is invalid.
```suggestion
                for ``aux_meas_op = Z-1j*Y`` we compute 2⟨(dω⟨ψ(ω)|)O(θ)|ψ(ω).
```